Tree.js

```diff
@@ -63,6 +63,7 @@ define([
                     layerData = _.assign({}, serviceLayer || {}, layerDetails || {}),
                     layerId = this.createLayerId(parent, layerData),
                     node = _.assign(layerData, {
+                        includeAllLayers: !!serviceLayer.subLayerIds,
```

Comment:
It isn't obvious from the code, but in this context, we are generating a `LayerNode` from dynamically loaded map service data. This function is only executed for "on demand" nodes that have no corresponding configuration in layers.json.

Since there is no configuration data for dynamically loaded nodes, when a group/folder node is encountered, the `includeAllLayers` property defaults to `false`. As a result, child nodes are not displayed.

What this code does is to provide a better default value for dynamically loaded group/folder nodes, so that child layers are rendered on the tree.

I'll add a comment. :smile: